wps_shared/wps_shared/wildfire_one/wildfire_fetchers.py

```diff
@@ -145,7 +145,7 @@ def prepare_fetch_hourlies_query(raw_station: dict, start_timestamp: datetime, e
 
     station_id = raw_station["id"]
     params = {"startTimestamp": query_start_timestamp, "endTimestamp": query_end_timestamp, "stationId": station_id}
-    endpoint = "/v1/hourlies/search/" "findHourliesByWeatherTimestampBetweenAndStationIdEqualsOrderByWeatherTimestampAsc"
```

Comment:
Strange quotes... I guess they weren't interpolated before so it worked fine?

Reply:
Yeah, I didn't know implicit concatenation was a thing! Fixed in https://github.com/bcgov/wps/pull/4271/commits/191d0966e0b7ac9318a3bb0027b2170f7e1d355c